data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
 			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
 			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
 			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`
+
+
+
+mission "James Medicine [0]"
+	minor
+	name "Medicine from <planet>"
+	description "James's doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."
```

Comment:
```suggestion
	description "James' doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."
```
doctors is plural, so this should be `James'`. [Ref](https://data.grammarbook.com/blog/apostrophes/apostrophes-with-names-ending-in-s-ch-or-z/)

Reply:
James is not plural for Jame. The number of a possessive depends on the owner, not on the owned thing.
Consider:
* The performer's instrument (a piano and a piano player)
* The performer's instruments (a piano, a violin, and a talented artist)
* The performers' instruments (a piano, a piano player, a violin, and a violin player)
* The performers' instrument (a piano and two piano players using it at once).

Reply:
Eh, no. It's James' doctor/s.
As long as you have an s before the single quote, no s is required at the back.

Reply:
To quote [grammarly](https://www.grammarly.com/blog/possessive-case/):
> Do we add another s for the possessive form of a name ending with s? Which is correct, Chris’s chair or Chris’ chair? James’s car or James’ car? Actually, both ways are correct. If a proper name ends with an s, you can add just the apostrophe or an apostrophe and an s.

Now, in the Endless Sky, there are cases of both. Consider:
* `Davy Jones' Locker`, from `human/names.txt` / `human/hails.txt`
* `Lieutenant Paris' voice` from `human/deep missions.txt`
* `Paris's voice` from `human/deep missions.txt`
* `Ijs's office` from `human/free worlds reconciliation.txt`
* `James's face` from `intro missions.txt`.

In general, my personal opinion is that `James's` is better than `James'`, but if someone pushes hard enough for it, I might change it :smiley:.